sw/device/tests/otbn_randomness_test.c

```diff
@@ -152,6 +154,61 @@ void initialize_clkmgr(void) {
   CLKMGR_TESTUTILS_CHECK_CLOCK_HINT(clkmgr, kOtbnClock, kDifToggleEnabled);
 }
 
+// Reinitialize both EDNs in auto-request (and non-boot) mode.
+static void reinit_edn_auto_mode(void) {
+  const dif_csrng_t csrng = {
+      .base_addr = mmio_region_from_addr(TOP_EARLGREY_CSRNG_BASE_ADDR)};
+  const dif_edn_t edn0 = {
+      .base_addr = mmio_region_from_addr(TOP_EARLGREY_EDN0_BASE_ADDR)};
+  const dif_edn_t edn1 = {
+      .base_addr = mmio_region_from_addr(TOP_EARLGREY_EDN1_BASE_ADDR)};
+
+  // Disable CSRNG.  This is required to abort the generate command of boot
+  // mode, which is likely still running.
+  CHECK_DIF_OK(dif_csrng_stop(&csrng));
+
+  // Stop both EDNs.
+  CHECK_DIF_OK(dif_edn_stop(&edn0));
+  CHECK_DIF_OK(dif_edn_stop(&edn1));
+
+  // Re-enable the CSRNG.
+  CHECK_DIF_OK(dif_csrng_configure(&csrng));
+
+  // Re-enable EDN0 in auto mode.
+  const dif_edn_auto_params_t edn0_params = {
+      // EDN0 provides lower-quality entropy.  Let one generate command return 8
+      // blocks, and reseed every 32 generates.
+      .reseed_material =
```

Comment:
Opinion only: for lower quality entropy, this seems like a high rate of reseeds - once per 256 128b chunks. I would perhaps pick 1 per thousands or millions. this is a good system level question.

Reply:
Thanks for pointing this system-level question on the lower-quality entropy network (EDN0) out, @mwbranstad.

WDYT, @msfschaffner @moidx, for EDN0, how many blocks should one generate command return and after how many generates should a reseed happen?

Reply:
for the test i was looking at with @engdoreis , i kept the first parameter "small".  Mostly because I wanted to create some stress on the csrng (so it wasn't just one command from `edn` to `csrng` during the whole test). 

The same argument I would apply to the second parameter, except doing so could introduce some instability to tests..since it potentially means based on how they are randomized the test could encounter much longer delays if the reseed is hit.